metrics/conflict.py

```diff
@@ -129,30 +126,34 @@ def __calculate_time_diffs(self, elegible_actions):
         # first calculate the times for all cases. This will produce some errors because
         # the shift is not aware of the tokens (revision times should belong to the same
         # token). This errors are removed in the next lines
-        df['time_diff'] = df['rev_time'] - df.shift(2)['rev_time']
-
+        
+        # changed:  instead of shifting by 2, shifting by 1
+        df['time_diff'] = 0
```

Comment:
this line is unnecessary, line 132 and 145 will override it anyway

Reply:
done